gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,58 @@ def get_possible_database_path():
     return path
 
 
+def create_possible_database_path():
```

Comment:
`create_possible_database_path()` -> `create_database_directory()`

The GRASS terminology here is all over the place here (grassdata, gisbase, ...), but "possible" does not make sense here and we are not creating a path, but the actual directory, so that's way "...database_directory".